website/assets/js/components/js-timestamp.component.js

```diff
@@ -20,7 +20,8 @@ parasails.registerComponent('jsTimestamp', {
   props: [
     'at',// « The JS timestamp to format
     'short',// « Whether to shorten the formatted date by not including the time of day (may only be used with timeago, and even then only applicable in certain situations)
-    'format',// « one of: 'calendar', 'timeago' (defaults to 'timeago'.  Otherwise, the "calendar" format displays data as US-style calendar dates with a four-character year, separated by dashes.  In other words: "MM-DD-YYYY")
+    'format',// « one of: 'timeago', 'calendar', 'billing' (defaults to 'timeago'.  Otherwise, the "calendar" format displays data as US-style calendar dates with a four-character year, separated by dashes.  In other words: "MM-DD-YYYY". The "billing" format displays timestamp in an expanded calendar format without the time of day, e.g. Dec 13, 2022)
+    'alwaysShowYear',// « (only for 'timeago') Whether to always include the year in the timestamp.
```

Comment:
the "only for 'timeago'" comment is confusing to me-- should it say "only for 'billing'"?